content/js2/blocks/response/index.md

```diff
@@ -11,46 +11,46 @@ emoji= '🧩'
 
 As a user, we interact with the elements on a web page. We _click_ on buttons, _input_ text, _submit_ forms etc.
 
-To react to an event, we can declare a function that we want to run whenever a certain event occurs. We call this function an **event handler**. In the example below, we _name_ this function `handleKeyup`:
```

Comment:
I want to specifically _not_ use `handleKeyup` as a name here, because people start thinking they're magical names (either that the name must be exactly that, or that they shouldn't think of better names because there's an "obvious" name).

Let's establish this practice early :)

Reply:
Absolutely!